packages/liveblocks-node/src/PermissionSet.ts

```diff
@@ -0,0 +1,146 @@
+// As defined in the source of truth in ApiScope in
+// https://github.com/liveblocks/liveblocks-cloudflare/blob/main/src/security.ts
+const ALL_PERMISSIONS = Object.freeze([
+  "room:write",
+  "room:read",
+  "room:presence:write",
+  "comments:write",
+  "comments:read",
+] as const);
+
+export type Permission = (typeof ALL_PERMISSIONS)[number];
+
+function isPermission(value: string): value is Permission {
+  return (ALL_PERMISSIONS as readonly unknown[]).includes(value);
+}
+
+const MAX_PERMS_PER_SET = 10;
+
+/**
+ * Assign this to a room or room pattern if you want to grant the user
+ * read-only permissions for this room.
+ */
+export const READ_ACCESS: readonly Permission[] = Object.freeze([
+  "room:read",
+  // "room:presence:write",  // XXX Should this be included in the READ_ACCESS set?
```

Comment:
@ofoucherot What do you think of this? Should we include the `room:presence:write` permission in the `READ_ACCESS` preset? This may be a bit confusing because it would mean there is a write permission included something called `READ_ACCESS`, but… I still think it would make for the best default. (Write access to your own sandbox aka Presence seems like the most useful default here.) We could add an alternative preset that is truly, completely, read-only, but I think the majority wouldn't use it and these presets try to optimize for the majority.

Reply:
> Write access to your own sandbox aka Presence seems like the most useful default here
I agree! It should be the default.